enclave-runner/src/usercalls/mod.rs

```diff
@@ -601,55 +834,52 @@ impl UsercallExtension for UsercallExtensionDefault{
 }
 
 #[allow(unused_variables)]
-impl RunningTcs {
-    fn entry(
+impl<'a> IOHandlerInput<'a> {
+
+    fn entry_async(
```

Comment:
This and coentry_async don't really make sense as a `fn` on `impl IOHandlerInput`. Instead, create a `fn do(io_send_queue: &mpsc::Sender<UsercallSendData>)` on `impl Work`.